addons/sale/wizard/mass_cancel_orders_wizard.py

```diff
@@ -0,0 +1,31 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
```

Comment:
```suggestion


```

nitpicking, but "Surround top-level function and class definitions with two blank lines." (PEP8)